xls/ir/BUILD

```diff
@@ -586,7 +586,7 @@ build_test(
 genrule(
     name = "op_header",
     outs = ["op.h"],
-    cmd = "$(location :render_specification_against_template) xls/ir/op_header.tmpl" +
+    cmd = "python $(location :render_specification_against_template) xls/ir/op_header.tmpl" +
```

Comment:
I was able to get the code running without the "python $(loca...." part. The advantage of having it is that with it I do not need to have the python3 setup custom path explained.

(It can work without this part)